### DISCUSSION TITLE
How to Name  Sections

### DISCUSSION THREAD

**Discussion by tech-ea** [excelerate-america/ea-features#233] [Best Practices]
### Proposal:
[feature]_[feature-type]_[page-type]_sec_{order in stack}

### EXAMPLES:
The section in figma that has the title enjoy perks and has picture?

service_odp_land_sec_03

**Comment by tech-ea** [comment]
@sorengrey, @youngmin-ea  I know we talked about something slightly different, but as I tried  to start naming it kind of broke down. So I added the above, tell me what you think

**Reply by sorengrey** [reply]
I think this section naming convention works well, but what would we call the blocks inside? I'm adding a `BaseBlockHeroService` to service_odp_land_sec_01. Should I call it service_odp_land_base_block_hero_service?
